[2024-12-05T12:19:34.514+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: P2M3.fetch-data manual__2024-12-05T12:18:59.835789+00:00 [queued]>
[2024-12-05T12:19:34.522+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: P2M3.fetch-data manual__2024-12-05T12:18:59.835789+00:00 [queued]>
[2024-12-05T12:19:34.522+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2024-12-05T12:19:34.523+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 1
[2024-12-05T12:19:34.523+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2024-12-05T12:19:34.534+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): fetch-data> on 2024-12-05 12:18:59.835789+00:00
[2024-12-05T12:19:34.539+0000] {standard_task_runner.py:52} INFO - Started process 220 to run task
[2024-12-05T12:19:34.541+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'P2M3', 'fetch-data', 'manual__2024-12-05T12:18:59.835789+00:00', '--job-id', '81', '--raw', '--subdir', 'DAGS_FOLDER/P2M3_alridho_DAG.py', '--cfg-path', '/tmp/tmptn05vkuy', '--error-file', '/tmp/tmpw81h978n']
[2024-12-05T12:19:34.542+0000] {standard_task_runner.py:80} INFO - Job 81: Subtask fetch-data
[2024-12-05T12:19:34.634+0000] {task_command.py:371} INFO - Running <TaskInstance: P2M3.fetch-data manual__2024-12-05T12:18:59.835789+00:00 [running]> on host f1a3a7cf14eb
[2024-12-05T12:19:34.752+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=alridho
AIRFLOW_CTX_DAG_ID=P2M3
AIRFLOW_CTX_TASK_ID=fetch-data
AIRFLOW_CTX_EXECUTION_DATE=2024-12-05T12:18:59.835789+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-12-05T12:18:59.835789+00:00
[2024-12-05T12:19:35.202+0000] {python.py:173} INFO - Done. Returned value was: None
[2024-12-05T12:19:35.228+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=P2M3, task_id=fetch-data, execution_date=20241205T121859, start_date=20241205T121934, end_date=20241205T121935
[2024-12-05T12:19:35.275+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-12-05T12:19:35.337+0000] {local_task_job.py:279} INFO - 1 downstream tasks scheduled from follow-on schedule check
